app/src/main/res/drawable/ic_preview_audio.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
WARNING: [UnusedResources] The resource `R.drawable.ic_preview_audio` appears to be unused
 Unused resources make applications larger and slow down builds.